docs/Agent/user-guide/man_page.md

```diff
@@ -1 +1,439 @@
-# Man Page
+# Man pages
+
+---
+
+### pbench-clear-results
+
+---
+
+**NAME**  
+pbench-clear-results, Clears the result directory
+
+**SYNOPSIS**  
+`pbench-clear-results [-C][--help]`
+
+**DESCRIPTION**  
+This command clears the results directory from /var/lib/pbench-agent directory
+
+**OPTIONS**  
+ -C, --config PATH  
+Path to a pbench-agent configuration file (defaults to the '\_PBENCH_AGENT_CONFIG' environment variable, if defined) [required]
+
+--help  
+Show this message and exit.
+
+---
+
+### pbench-clear-tools
+
+---
+
+**NAME**  
+pbench-clear-tools, Clear registered tools by name or group
+
+**SYNOPSIS**  
+`pbench-clear-tools [-C][-n][-g][-r][--help]
+`
+
+**DESCRIPTION**  
+Clear all tools which are registered and can filter by name of the group
+
+**OPTIONS**  
+-C, --config PATH  
+Path to a pbench-agent configuration file (defaults to the '\_PBENCH_AGENT_CONFIG' environment variable, if defined) [required]
+
+-n, --name,--names STR  
+Clear only the <tool-name> tool.
+
+-g, --group,--groups STR  
+Clear the tools in the <group-name> group. If no group is specified, the 'default' group is assumed.
+
+-r, --remote, --remotes STR  
+Clear the tool(s) only on the specified remote(s). Multiple remotes may be specified as a comma-separated list. If no remote is specified, all remotes are cleared
+
+--help  
+Show this message and exit.
+
+---
+
+### pbench-copy-results
+
+---
+
+**NAME**  
+pbench-copy-results, Copies result tarball to the server
+
+**SYNOPSIS**  
+`pbench-copy-results [--help] --user=<user> [--controller=<controller>] [--prefix=<path>][--xz-single-threaded] [--show-server]
+`
+
+**DESCRIPTION**  
+It keeps the copy of tarball on the pbench-agent and pushes the other to pbench-server
```

Comment:
"the other" is not really a good way to state this. Maybe something like "Copy result tarballs to a 0.69 Pbench Server without deleting the local tarballs."

Reply:
"on the pbench-agent" isn't a good phrase, either.  "on the host where this command is run" might be better (or "on the local host").

And, actually, what this command does is to push the benchmark result to the Pbench Server **_without_** removing it from the local host -- it doesn't so much create a local copy of anything.  (E.g., this man page doesn't say anything about where the local "copy of tarball" can be found....)

And, to be honest, I think it would be good if we avoided talking about `tar` and "tarballs" as much as possible.  The use of `tar` is largely hidden from the user, and I see no reason to document it.